services/notifications/src/modules/mail/enums/template-subject.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum TemplateSubjectEnum {
+  test = 'Ecstasify🎵: Test email',
+  'user-registered' = 'Ecstasify🎵: Welcome',
```

Comment:
Why not to use `UserRegistered`? Any speacial reason for that?

Reply:
Because I use enum keys equal to template name to match template name and subject